ndarray/src/main/java/org/tensorflow/ndarray/StdArrays.java

```diff
@@ -3798,11 +3796,12 @@ private static void dimSize(int arrayLength, long[] shape, int dimIdx) {
     }
     int[] arrayShape = new int[expectedRank];
     for (int i = 0; i < expectedRank; ++i) {
-      long dimSize = shape.size(i);
+      long dimSize = shape.get(i);
```

Comment:
There are a huge number of formatting changes in this file for this two line update. Could you revert everything bar the necessary change?